modules/ROOT/pages/anypoint-mq/2.x/anypoint-mq-listener.adoc

```diff
@@ -230,6 +230,31 @@ In many cases there's a single common service between the processing of messages
 
 For this scenario, both subscribers stop polling for messages as soon as the error count reaches the `errorsThreshold="10"` value, counting both `FTP:RETRY_EXHAUSTED` and `HTTP:SERVICE_UNAVAILABLE` errors. When the `tripTimeout` elapses, one of the subscribers polls for a message and uses it to test the circuit, enabling the polling for both subscribers if the processing of that message succeeds.
 
+
+== FIFO Queues
+
+FIFO queues are most suitable for single-consumer scenarios. When one consumer is accessing a message, all other consumers are blocked until the first batch is processed. No messages are delivered until all in-flight messages are acknowledged or not acknowledged. With message groups, multiple consumers can access messages in a FIFO queue. In this case, a single consumer accesses messages in a group and message order is preserved within the message group.
+
+----
+IMPORTANT: 
+Always use a single-thread flow configuration when consuming messages from a FIFO Queue. If flow processing is not single-threaded, messages processing order might be lost.
+----
+
+=== FIFO Queues and Clustering  
+
+FIFO queues behave the same in a clustered environment as in a non-clustered environment.
+
+FIFO queues consume messages in the specified order. After the message is consumed, any further message processing can be distributed to other nodes. In this case, if the consumer acknowledges a message before it is fully processed, message order might be lost during message processing.
+
+* In an xref:mule-runtime::mule-high-availability-ha-clusters.adoc[on-prem, high availability clustering environment], the Anypoint MQ connector Subscriber source runs on all nodes by default. 
```

Comment:
on-premises, high-availability environment,

Reply:
Does "connector" modify Anypoint MQ or subscriber source? 
Do we have to cap subscriber?